Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -38,8 +39,13 @@ using tstring = std::string;
 
 namespace vtStor
 {
-	using DeviceHandle = int;
-	using HANDLE = int;
+
+struct DeviceHandle
```

Comment:
This define use to avoid static storage on StorageUtility. In order to find bus from DeviceHandle i created the structure  to make a associate for DeviceHandle and AdapterProperty. On Window os, We have support of API for find Adapter property from Device Handle, but Linux os can not support.
This define lead to asynchonous DeviceHandle (with cross-platform).
We can fix issue by define common DeviceHandle for multi os or finding support of API Linux for get AdapterProperty from Handle.